app/assets/stylesheets/_global.scss

```diff
@@ -9,5 +9,40 @@ body {
 }
 
 .nav-bar {
-  position: relative;
+  margin-top: .75em;
+  text-align: right;
+}
+
+#page-header {
+  background-color: #f8f8f8;
+  background-image: -webkit-linear-gradient(top, white 0%, #f8f8f8 100%);
+  background-image: -o-linear-gradient(top, white 0%, #f8f8f8 100%);
+  background-image: linear-gradient(to bottom, white 0%, #f8f8f8 100%);
+  background-repeat: repeat-x;
+  filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#FFFFFFFF', endColorstr='#FFF8F8F8', GradientType=0);
+  filter: progid:DXImageTransform.Microsoft.gradient(enabled = false);
+  border-radius: 4px;
+  -webkit-box-shadow: inset 0 1px 0 rgba(255, 255, 255, 0.15), 0 1px 5px rgba(0, 0, 0, 0.075);
+  box-shadow: inset 0 1px 0 rgba(255, 255, 255, 0.15), 0 1px 5px rgba(0, 0, 0, 0.075);
+  border: 1px solid #e7e7e7;
+}
+
+.nav {
+  float: none;
+  width: 100%;
```

Comment:
100% is default for any block level item and there's no other style overriding thing, so we can drop it.